framework/test/AnyTest.cpp

```diff
@@ -29,6 +29,14 @@
 
 using namespace cppmicroservices;
 
+template <typename T>
+void TestUnsafeAnyCast(Any& anyObj, T val)
+{
```

Comment:
Maybe assign the unsafe cast to a local ptr?
```cpp
T* ptr = unsafe_any_cast<T>(&anyObj);
US_TEST_CONDITION(ptr, "unsafe_any_cast");
```
and similarly for other 2 `US_TEST_CONDITION`s